lib/ast/node.rb

```diff
@@ -177,9 +177,7 @@ def append(element)
     #     p value    # => (integer 1)
     #
     # @return [Array]
-    def to_a
-      children
-    end
+    alias to_a children
```

Comment:
Hmm, the docs is broken... `to_a` section doesn't exist in docs that is generated by yard.

I think the docs for `to_a` should be moved to the above of `children` definition. That way, the docs will be repaired.

What do you think?